tests/update_reference_data.sh

```diff
@@ -55,6 +55,21 @@ for test_name in ${tests}; do
       EGRID INIT SMSPEC UNRST UNSMRY
   fi
 
+  if grep -q "spe12p" <<< $test_name
```

Comment:
if update for this test is requested, copy output data from test to reference dir. parameters should be self-explanatory, mostly c&p and adjustments when adding a new one.